src/test/java/org/jenkinsci/backend/ircbot/IrcListenerTest.java

```diff
@@ -1,45 +1,41 @@
 package org.jenkinsci.backend.ircbot;
 
 import com.google.common.collect.ImmutableSortedSet;
-import junit.framework.TestCase;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Test;
 import org.kohsuke.github.*;
 import org.mockito.Mockito;
+import org.mockito.MockedStatic;
 import org.mockito.invocation.InvocationOnMock;
 import org.mockito.stubbing.Answer;
 import org.pircbotx.Channel;
 import org.pircbotx.User;
 import org.pircbotx.UserLevel;
 import org.pircbotx.output.OutputChannel;
-import org.powermock.api.mockito.PowerMockito;
-import org.powermock.core.classloader.annotations.PowerMockIgnore;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
 
 import static java.util.Collections.emptyList;
-import static org.mockito.Matchers.any;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 
 /**
  * Created by slide on 11/14/2016.
  */
-@RunWith(PowerMockRunner.class)
-@PrepareForTest(GitHub.class)
-@PowerMockIgnore("javax.net.ssl.*")
-public class IrcListenerTest extends TestCase {
+public class IrcListenerTest {
+    static MockedStatic<GitHub> utilities = Mockito.mockStatic(GitHub.class);
```

Comment:
does this need closing after the test class is completed to not leak resources / affect other tests?

Reply:
None of the other tests currently use GitHub stuff, but it would be good practice I guess.